### PR TITLE
feat: accordion component

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**
- adds new accordion component.

### CODE REVIEW

File: src/components/organisms/accordion/index.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+//       })}
+//     >
+//       {children}
+//     </div>
+//   </div>
+// </div>
+//   )
+// }
+
+// Accordion.Item = Item
```
Comment (zakariaelas) [inline src/components/organisms/accordion/index.tsx:79]: rm maybe?

File: src/components/organisms/accordion/index.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  description,
+  required,
+  tooltip,
+  children,
+  ...props
+}) => {
+  return (
+    <AccordionPrimitive.Item
+      {...props}
+      className="border-b border-grey-20 transition- pb-5 radix-state-open:pb-5xlarge mb-5 last:mb-0 group"
```
Comment (zakariaelas) [inline src/components/organisms/accordion/index.tsx:112]: looks like the `transition` class is missing some suffix maybe?

File: src/components/organisms/accordion/index.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+const MorphingTrigger = () => {
+  return (
+    <Button variant="ghost" size="small" className="p-[6px] relative group">
+      <div className="w-5 h-5">
+        <span className="absolute bg-grey-50 rounded-circle inset-y-[31.75%] left-[48%] right-1/2 w-[1.5px] group-radix-state-open:rotate-90 duration-300" />
+        <span className="absolute bg-grey-50 rounded-circle inset-x-[31.75%] top-[48%] bottom-1/2 h-[1.5px] group-radix-state-open:rotate-90 group-radix-state-open:left-1/2 group-radix-state-open:right-1/2 duration-300" />
+      </div>
+    </Button>
+  )
+}
```
Comment (zakariaelas) [inline src/components/organisms/accordion/index.tsx:147]: love this 🤩 !!

**Review by zakariaelas — APPROVED** [review]
Great work Kasper! Really love the animations 😍 

Small nit: it looks like the animation is different between the two accordion items in storybook. The first one is a slide down/slide up when opening/closing the accordion, whereas the second item looks more like a fade in/fade out. Is it just me or do you see it too 😅 ?

File: src/components/organisms/accordion/index.tsx (hunk 3 above)
Comment (pKorsholm) [reply src/components/organisms/accordion/index.tsx:147]: Agree, love the animation 😍

**Review by pKorsholm — APPROVED** [review]
LGTM1 
This is very cool, the animations on all transitions gives the component a great feel!

same nit as Zak. 

Since this doesn't affect functionality, we should consider opening a new ticket if we need to get this in for next sprint so work can continue on price lists in admin.

**Comment by srindom** [comment]
@zakariaelas @pKorsholm can one of you open a ticket for the nits so we can merge for now? :)

**Pull request merged**